### PR TITLE
add export docs

### PR DESCRIPTION
**Pull request by xin3he**
Signed-off-by: Xin He <xin3.he@intel.com>

## Type of Change

add document of export API.

### CODE REVIEW

**Review by WenjiaoYue — COMMENTED** [review]
Open Neural Network Exchange (ONNX)

**Review by lvliang-intel — APPROVED** [review]
LGTM.

File: docs/source/export.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+1. [Introduction](#introduction)
+
+2. [Supported Framework Model Matrix](#supported-framework-model-matrix)
+
+3. [Examples](#examples)
+
+4. [Appendix](#appendix)
+
+# Introduction
+Open Neural Network eXchange (ONNX) is an open standard format for representing machine learning models. Exporting FP32 PyTorch/Tensorflow models has become popular and easy to use. However, for Intel Neural Compressor, we hope to export the int8 model to ONNX to achieve higher applicability on multiple frameworks.
```
Comment (XuhuiRen) [inline docs/source/export.md:13]: export INT8 model to the ONNX format

Comment (xin3he) [reply docs/source/export.md:13]: okay

Comment (XuhuiRen) [reply docs/source/export.md:13]: should be "in multiple frameworks"?

File: docs/source/export.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+Here we briefly introduce our export API for PyTorch FP32/INT8 models. First, the int8 ONNX model is not directly exported from the int8 PyTorch model, but quantized after obtaining the FP32 ONNX model using the mature torch.onnx.export API. To ensure that the quantization process of ONNX is as consistent as possible with PyTorch, we reuse three key pieces of information from the Neural Compressor model to perform ONNX quantization.
```
Comment (XuhuiRen) [inline docs/source/export.md:15]: all "int8" should be changed to "INT8"

Comment (XuhuiRen) [inline docs/source/export.md:15]: should be "is consistent with"

File: docs/source/export.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+ - Quantized operations: Only operations quantized in PyTorch will be quantized in ONNX.
```
Comment (XuhuiRen) [inline docs/source/export.md:17]: "will be quantized into ONNX format"?

File: docs/source/export.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+ - Scale info: Scale information is collected from the PyTorch quantization process.
+ - Weights of quantization aware training(QAT): for quantization-aware training, the updated weights are passed to the ONNX model.
```
Comment (XuhuiRen) [inline docs/source/export.md:19]: select a consistent name "quantization aware training" or "quantization-aware training"

File: docs/source/export.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+   QuantizeLinear
+         |
+  DequantizeLinear
+</pre>
+     </td>
+   </tr>
+ </tbody>
+</table>
+
+The default recipe is `QDQ_OP_FP32_BIAS`. If the accuracy of the exported ONNX INT8 model does not meet your criterion, we recommend you try recipe `QDQ_OP_INT32_BIAS` and `QDQ_OP_FP32_BIAS_QDQ` as follows:
```
Comment (XuhuiRen) [inline docs/source/export.md:173]: "does not" should be changed to "cannot"

File: docs/source/export.md (hunk 2 above)
Comment (xin3he) [reply docs/source/export.md:15]: Actually, they are not consistent in all patterns.

**Review by WenjiaoYue — APPROVED** [review]

**Pull request merged**